src/tests/Loader/classloader/MethodImpl/CovariantReturns/Interfaces/UnitTest.il

```diff
@@ -321,7 +321,7 @@
     throw
   }
 
-  .method public hidebysig virtual instance class IGenRetType<!0, class IDictionary<string,!1>> MyGenFunc(string& res)
+  .method public hidebysig virtual instance class IGenRetType<!2, class IDictionary<string,!3>> MyGenFunc(string& res)
```

Comment:
@janvorli could you have a look at the covariant IL test changes? I believe these were test bugs - they were not overriding methods from the base because the base is not even instantiated over !0 and !1 (they're called UNUSED).

This was asserting the NativeAOT compiler. Maybe it shouldn't assert it, but that's another thing. The conditions are really obscure.